### PR TITLE
Upgrading preprod server documentation

### PR DESCRIPTION
**Pull request by mohamedzayed92**

### CODE REVIEW

File: doc/preprod_update.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+
+This brings the entire dependency up to latest.
+However, for ngnix, you have to copy/paste the static folders to a volumne that can be shared with other containers.
+
+Nginx container::
+
+	cd /opt/assembl_static
+	./update.sh
+	rm - rf static_backup
```
Comment (swergas) [inline doc/preprod_update.rst]: remove space

File: doc/preprod_update.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
+Updating the preprod Server
+===========================
+
+This method is a temporary method in order to quickly update the preprod server with the latest version of the code.
+
+Preprod server has an assembl instance running on different containers. The whole instance is run via a .yml file run with docker-compose
+
+
+Note: 6f41608ee826 is the identity of the container running the image assembl:develop
```
Comment (ayyazdaniaryan) [inline doc/preprod_update.rst]: I wouldn't do this. The next time the image is rebuilt, the docs will have to change to represent this new container ID. I would only describe that the container id can be found using `docker ps | grep assembl`

File: doc/preprod_update.rst (hunk 1 above)
Comment (ayyazdaniaryan) [reply doc/preprod_update.rst]: Yup, no space after dash. `rm -rf static_backup`

File: doc/preprod_update.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+This brings the entire dependency up to latest.
+However, for ngnix, you have to copy/paste the static folders to a volumne that can be shared with other containers.
+
+Nginx container::
+
+	cd /opt/assembl_static
+	./update.sh
+	rm - rf static_backup
+	mv static static_backup2
+	mv static2 static_backup2
```
Comment (ayyazdaniaryan) [inline doc/preprod_update.rst]: I think now the shell script manages this as well. So you can remove this line and the one above.

File: assembl/static2/js/app/components/administration/exportSection.jsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
+// @flow
```
Comment (cedricmessiant) [inline assembl/static2/js/app/components/administration/exportSection.jsx]: why this file is in this commit ? Maybe a bad merge ?

**Pull request closed without merging**